src/Classes/Container.ts

```diff
@@ -208,24 +208,33 @@ export class Container {
         return JSON.stringify(this.options, undefined, 2);
     }
 
-    public async loadImage(image: IImage, optionsImage: ImageShape): Promise<void> {
-        image.error = false;
+    public loadImage(image: IImage, optionsImage: ImageShape): Promise<void> {
+        return new Promise((resolve: (value?: void | PromiseLike<void> | undefined) => void,
+            reject: (reson?: any) => void) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5264055)